src/config/config.c

```diff
@@ -656,6 +656,37 @@ __wt_config_getones_none(
     return (0);
 }
 
+/*
+ * __wt_config_tokenizer --
+ *     String tokenizer into the given array.
+ */
+int
+__wt_config_tokenizer(
+  WT_SESSION_IMPL *session, const char *cval_str, const char *delimeter, uint64_t *parse_array)
+{
+    uint64_t snap_value;
```

Comment:
This function can be called from anyone in future, so better to rename the variable to `value` and `counter`.

Reply:
Fixed. Updated the function with generic variable names.